vars/buildWDKAutoSwitch.groovy

```diff
@@ -146,6 +149,16 @@ def call(Map config = [unityVersions:[]]) {
                     }
                   }
 
+                  def catchStep(Exception e) = {
```

Comment:
To create a closure bound to a variable you need to write it like this:
```suggestion
                  def catchStep = { Exception e ->
```

Reply:
Thank you.